module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1394,18 +1397,36 @@ public function renewMyItems($renewDetails)
      */
     public function getPickupLocations($patron, $holdInfo = null)
     {
+        $limitedServicePoints = false;
```

Comment:
I wonder if null would be a better initialized value for this. Shouldn't change the logic below, but mixing null with an array feels cleaner than mixing a boolean false with an array.

Reply:
Agreed, done.